docs/root/configuration/http/http_filters/_include/credential-injector-oauth2-filter.yaml

```diff
@@ -0,0 +1,80 @@
+static_resources:
+  listeners:
+  - name: listener_0
+    address:
+      socket_address:
+        protocol: TCP
```

Comment:
nit: do we need to specify protocol here ?

Reply:
not required. Will remove. Thanks!